scripts/bin/GenerateStaticData.pm

```diff
@@ -427,7 +427,9 @@ sub _texts {
         # Strings for lost_password
         push @texts, localize('[_1] Please click the link below to restart the password recovery process.');
         push @texts, localize('Your password has been successfully reset. Please log into your account using your new password.');
-        push @texts, localize('Please check your email to complete the process.');
+        push @texts, localize('Please check your email for the verification link to complete the process.');
+        push @texts, localize('If you don’t receive the email within the next few minutes, please check your junk/spam folder.');
+        push @texts, localize('To reset your password, enter the email address you used to create your account into the field below and click \'Reset password\'.');
```

Comment:
why do we need to add these three strings to this file? as per this PR they are only being used in templates and no .js files, or am I wrong?

Reply:
ah yes! it's used in templates only